### PR TITLE
Add --universal to mac builds

### PR DESCRIPTION
**Pull request by nitrocode**
Closes https://github.com/bitwarden/desktop/issues/792

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,9 +37,9 @@
     "clean:l10n": "git push origin --delete l10n_master",
     "pack:dir": "npm run clean:dist && electron-builder --dir -p never",
     "pack:lin": "npm run clean:dist && electron-builder --linux --x64 -p never",
-    "pack:mac": "npm run clean:dist && electron-builder --mac -p never",
-    "pack:mac:mas": "npm run clean:dist && electron-builder --mac mas -p never",
-    "pack:mac:masdev": "npm run clean:dist && electron-builder --mac mas-dev -p never",
+    "pack:mac": "npm run clean:dist && electron-builder --mac --universal -p never",
```
Comment (Hinton) [inline package.json:40]: This unfortunately doubles the application size, we can't do much about the Mac App Store unfortunately but for `dmg` we should strive towards creating two separate binaries which the users may download.

Comment (nitrocode) [reply package.json]: Thank you for the comment. I added a `pack:mac_arm64` target and reverted the `pack:mac` target.

I left the `--universal` target for the Mac App Store.

File: package.json [hunk 2]
```diff
@@ -38,8 +38,9 @@
     "pack:dir": "npm run clean:dist && electron-builder --dir -p never",
     "pack:lin": "npm run clean:dist && electron-builder --linux --x64 -p never",
     "pack:mac": "npm run clean:dist && electron-builder --mac -p never",
-    "pack:mac:mas": "npm run clean:dist && electron-builder --mac mas -p never",
-    "pack:mac:masdev": "npm run clean:dist && electron-builder --mac mas-dev -p never",
+    "pack:mac_arm64": "npm run clean:dist && electron-builder --mac --arm64 -p never",
```
Comment (Hinton) [inline package.json:41]: Nitpick but for the other jobs we use `:`, not `_`. @joseph-flinn Something you could fix while hooking up the CI jobs perhaps?

**Review by Hinton — APPROVED** [review]

**Comment by Hinton** [comment]
Looks good, thanks for the contribution @nitrocode.

File: package.json (hunk 2 above)
Comment (cscharf) [reply package.json:41]: I would prefer we keep consistency here across the board. @nitrocode , can you please fix this syntax?
```suggestion
    "pack:mac:arm64": "npm run clean:dist && electron-builder --mac --arm64 -p never",
```

**Review by cscharf — CHANGES_REQUESTED** [review]

**Comment by nitrocode** [comment]
@Hinton I kept the target consistent using @cscharf suggestion. Thank you!

**Review by cscharf — APPROVED** [review]
Thank you!

**Pull request merged**